devel/meson/Portfile

```diff
@@ -21,9 +19,14 @@ long_description    Meson  is  a  build system designed to optimize programmer p
                     Valgrind,  CCache  and  the like. It is both extremely fast, and, even more importantly, \
                     as user friendly as possible.
 
-checksums           rmd160  61f9437b9c405be9d6dd4fe53d2ad5657fa14909 \
-                    sha256  92d8afd921751261e36151643464efd3394162f69efbe8cd53e0a66b1cf395eb \
-                    size    1267981
+homepage            https://mesonbuild.com/
+
+master_sites        pypi:m/${name}
```

Comment:
What was wrong with github?

Anyway, this port is a mixture of update, switch to pypi (ok, it makes sense if the two are done together) and unrelated rearrangement of code with no functional changes. It's not absolutely necessary, but we try to decouple the "whitespace" changes from functional changes into two separate commits. But before that I'm really curious what the benefits of pypi would be in the first place